cyclops-ui/src/components/pages/NodeDetails.tsx

```diff
@@ -376,6 +396,14 @@ const NodeDetails = () => {
       <Row>
         <Title>{nodeName}</Title>
       </Row>
+      <Row>
+        <Text keyboard>
+          Created on:-{" "}
```

Comment:
Can you just remove the dash
```suggestion
          Created on: {" "}
```

Reply:
sorry, forgot to push that, 
https://github.com/cyclops-ui/cyclops/pull/443/commits/1573acc03b28498c089fc3c75eef36b82bbd523f#diff-3e0644bb482bb090ea078cc63a4529082f10f7215d27fdadd145afc6128afab3R401